app/views/repositories/new.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="page-header">
-  <h1>New Repository</h1>
+  <h1><%= "#{t('new.male')} #{Repository.model_name.human}" %></h1>
```

Comment:
We are going to discuss this soon. I do agree with you but fixing this will be a history of its own.